openwisp_controller/config/tests/test_vpn.py

```diff
@@ -239,7 +241,7 @@ def test_get_context(self):
     def test_dh(self, mocked_dhparam):
         mocked_dhparam.return_value = self._dh
         v = self._create_vpn()
-        v.dh = None
+        v.dh = ''
```

Comment:
is it really necessary to change this line?

Reply:
Since we are moving save to the last line, this change won't be necessary anymore. I can undo it